.travis.yml

```diff
@@ -0,0 +1,29 @@
+---
+language: ruby
+sudo: required
+cache: bundler
+rvm:
+- 2.4.5
+services:
+- docker
+addons:
+  postgresql: '9.5'
+env:
+  global:
+  - DATABASE_USERNAME=root
+  - DATABASE_PASSWORD=smartvm
+before_install:
+- gem install bundler -v 1.16.4
```

Comment:
Do we always want this version of bundler? I'll tell you right now that I'm going to forget to update this.

Reply:
I removed that and then put it back...I agree with you...let's just drop the version and install latest.